src/ios/hooks/ab/add-targets.js

```diff
@@ -276,4 +264,36 @@ function getUuidByComment(comment, section) {
     }
 
     return null;
+}
+
+function calculateFilePaths(dirname, filenames, excludingExtensions, excludingFiles) {
```

Comment:
We can combine the excludingExtensions and excludingFiles in a single array with excluding patterns